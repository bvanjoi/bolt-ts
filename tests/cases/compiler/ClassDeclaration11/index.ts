// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/ClassDeclaration11.ts`, Apache-2.0 License

class C {
   constructor(); //~ ERROR: Constructor implementation is missing.
   foo() { }
}