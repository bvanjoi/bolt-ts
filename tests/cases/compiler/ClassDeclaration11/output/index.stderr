  × Constructor implementation is missing.
   ╭─[../../tests/cases/compiler/ClassDeclaration11/index.ts:4:4]
 4 │ constructor
   · ───────────
   ╰────
